src/modules/common/containers/LanguageNotAvailableContainer.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import { connect } from 'react-redux'
+import type { Dispatch } from 'redux'
+import type { StateType } from '../../../modules/app/StateType'
+import type { StoreActionType } from '../../../modules/app/StoreActionType'
+import LanguageNotAvailablePage from '../../../modules/common/components/LanguageNotAvailablePage'
+import { withTheme } from 'styled-components/native'
+import compose from 'lodash/fp/compose'
+import { translate } from 'react-i18next'
+import createNavigateToCategory from '../../app/createNavigateToCategory'
```

Comment:
I suggest to keep imports in the following order:

1. packages
2. components
3. actions
4. constants
5. flow types
It increases readability and allows you to faster grasp what's imported.
```
import compose from 'lodash/fp/compose'
import { connect } from 'react-redux'
import { translate } from 'react-i18next'

import LanguageNotAvailablePage from '../../../modules/common/components/LanguageNotAvailablePage'
import { withTheme } from 'styled-components/native'
import createNavigateToCategory from '../../app/createNavigateToCategory'

import type { Dispatch } from 'redux'
import type { StateType } from '../../../modules/app/StateType'
import type { StoreActionType } from '../../../modules/app/StoreActionType'
```

Reply:
Would be a separate task imo since we don't do this anywhere yet, feel free to create a new task.